Rename gameName to appName

robert-altom opened the pull request.

robert-altom commented:
Ping @RobertPoienar @gomboskriszta15 @ReichEdvin

RobertPoienar commented on Assets/AltTester/AltDriver/AltDriver.cs:
```diff
@@ -22,13 +22,13 @@ public class AltDriver
         /// <param name="enableLogging">If true it enables driver commands logging to log file and Unity.</param>
         /// <param name="connectTimeout">The connect timeout in seconds.</param>
-        public AltDriver(string host = "127.0.0.1", int port = 13000, bool enableLogging = false, int connectTimeout = 60, string gameName = "__default__")
+        public AltDriver(string host = "127.0.0.1", int port = 13000, bool enableLogging = false, int connectTimeout = 60, string appName = "__default__")
```
Can you please document the appName parameter

RobertPoienar reviewed (changes requested):
Just a little comment, the rest looks ok

iulianatodoran commented on Assets/Editor/CreateAltPrefab.cs:
```diff
@@ -658,10 +222,524 @@ private static bool quaternionEquality(Quaternion originalQuaternion, Quaternion
+            LabelRectTransform.pivot = new Vector2(0.5f, 0.5f);
+
+            var LabelText = LabelGameObject.GetComponent<Text>();
+            LabelText.text = "To change the host, port or game name input a new value and press the Restart button.";
```
should be `app name` here

robert-altom replied:
Fixed.

iulianatodoran commented:
In the `AltDesktopPage.cs` there are 2 references to the `game`, maybe they should be also changed with `app`
![image](https://user-images.githubusercontent.com/62327834/221793090-74dbdb37-072f-4134-a4b1-37746c6b8e1f.png)

iulianatodoran commented:
All the screenshots with the `game` and `AltProxy` name need to be updated also

iulianatodoran commented:
Found that in all these files there are references to the game. should they also be changed?
![image](https://user-images.githubusercontent.com/62327834/221803910-896c4cf5-d66e-4fe1-ab11-a50cc13749db.png)

robert-altom commented:
> Found that in all these files there are references to the game. should they also be changed? ![image](https://user-images.githubusercontent.com/62327834/221803910-896c4cf5-d66e-4fe1-ab11-a50cc13749db.png)

I fixed the ones from docstings and `README.md`.

robert-altom commented:
> All the screenshots in the documentation with the `game` and `AltProxy` name need to be updated also

We also need to uncomment and update all references to `AltProxy`.

robert-altom commented:
> In the `AltDesktopPage.cs` there are 2 references to the `game`, maybe they should be also changed with `app` ![image](https://user-images.githubusercontent.com/62327834/221793090-74dbdb37-072f-4134-a4b1-37746c6b8e1f.png)

Fixed.
